udata/api/__init__.py

```diff
@@ -128,6 +130,7 @@ def page_parser(self):
                             help='The page size to fetch')
         return parser
 
+    cache_page = cache_page
```

Comment:
To be able to decorate with `api.cache_page` likewise other api-related decorators.